### PR TITLE
Renamed NewCollection to NeedsTriageCollection

### PR DESCRIPTION
**Pull request by haseebgit**
r? @miketaylr

### CODE REVIEW

File: webcompat/static/js/lib/diagnose.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
 var diagnose = diagnose || {}; // eslint-disable-line no-use-before-define
 var issues = issues || {}; // eslint-disable-line no-use-before-define
 
-diagnose.NewCollection = Backbone.Collection.extend({
+diagnose.NeedsTriageCollection = Backbone.Collection.extend({
   model: issues.Issue,
   url: '/api/issues/category/needstriage'
 });
```
Comment (miketaylr) [inline webcompat/static/js/lib/diagnose.js]: Looks good! Just a few more places to change:

https://github.com/haseebgit/webcompat.com/blob/6e5a553850968d89eb429d6c454e22a49c6d4bf4/webcompat/static/js/lib/diagnose.js#L13

https://github.com/haseebgit/webcompat.com/blob/6e5a553850968d89eb429d6c454e22a49c6d4bf4/webcompat/static/js/lib/diagnose.js#L18

https://github.com/haseebgit/webcompat.com/blob/6e5a553850968d89eb429d6c454e22a49c6d4bf4/webcompat/static/js/lib/diagnose.js#L34

https://github.com/haseebgit/webcompat.com/blob/6e5a553850968d89eb429d6c454e22a49c6d4bf4/webcompat/static/js/lib/diagnose.js#L23

https://github.com/haseebgit/webcompat.com/blob/6e5a553850968d89eb429d6c454e22a49c6d4bf4/webcompat/templates/home-page/browse-issues.html#L5

**Comment by miketaylr** [comment]
Can you also update the commit message according to the contribution guidelines?

https://github.com/webcompat/webcompat.com/blob/master/CONTRIBUTING.md#submission-guidelines

Thanks!

**Comment by miketaylr** [comment]
@haseebgit let me know if you need any help with this! ❤️

**Comment by haseebgit** [comment]
@miketaylr I have some doubts, I will ping you on IRC

**Comment by miketaylr** [comment]
> @miketaylr I have some doubts, I will ping you on IRC

Sure thing, feel free to ask questions here in the issue as well.

**Comment by haseebgit** [comment]
@miketaylr i edited this through web and now finding it difficult to update the PR, should i delete this branch and create new PR from my machine?

**Comment by miketaylr** [comment]
@haseebgit as long as you're editing the files on the "patch-1" branch, it should be possible from the web (and should automatically update the PR). 

(But it might be good to make edits locally, to verify that it still works after you make the changes. Let me know if you need help getting it running.)

**Comment by haseebgit** [comment]
@miketaylr r? 
there were many errors locally with git, edited on web.
I will follow proper guidelines for next patch :)

File: webcompat/templates/home-page/browse-issues.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <div class="wc-UIContent">
     <h2 class="wc-Title wc-Title--l">Latest Untriaged Issues</h2>
     <div id="js-lastIssue">
-      <script type="text/template" id="new-tmpl">
+      <script type="text/template" id="needstriage-tmpl">
         <% if (newIssues.length) { %>
```
Comment (miketaylr) [inline webcompat/templates/home-page/browse-issues.html]: Can we make this just `issues` instead of `newIssues`?

Comment (miketaylr) [inline webcompat/templates/home-page/browse-issues.html]: Actually, I can do that.

**Comment by miketaylr** [comment]
@haseebgit check out this article on syncing your fork with upstream, your current repo is out of date.

**Comment by miketaylr** [comment]
Thanks!

**Pull request merged**